Source/UIScrollView+EmptyDataSet.m

```diff
@@ -549,7 +549,8 @@ - (void)dzn_invalidate
 void dzn_original_implementation(id self, SEL _cmd)
 {
     // Fetch original implementation from lookup table
-    NSString *key = dzn_implementationKey(self, _cmd);
+    Class baseClass = dzn_baseClassToSwizzleForTarget(self);
+    NSString *key = dzn_implementationKey(baseClass, _cmd);
```

Comment:
@dzenbot This is the big difference between the old/new PR. The old one was still using `self`, which is what was causing the issue for subclasses.